mongo/integration/primary_stepdown_test.go

```diff
@@ -100,7 +100,7 @@ func isPoolCleared() bool {
 }
 
 func clearPoolChan() {
-	for len(poolChan) < 0 {
+	for len(poolChan) > 0 {
```

Comment:
Hmm was this just not working before?

Reply:
Yeah, `clearPoolChan` was a no-op before. The tests seem to not require it to work because they pass either way. I guess we could technically remove the function completely.

Reply:
Ah if it's truly a no-op, might as well remove it.

Reply:
Thinking about this more, I believe the `clearPoolChan()` function may help prevent tests from affecting each other when they fail (e.g. if a pool publishes more than the expected number of events and the test doesn't read all of them), so I'm going to leave it.